### PR TITLE
Updated tutorial.md

### PR DESCRIPTION
**Pull request by ktzee**
**- What I did**
Followed the tutorial. Had some issues. Modified the doc adding the steps I've taken to progress.
"crictl create" was not working with the image path in container_redis.json so I changed it to the path that made it work for me.
**- How I did it**
Followed the tutorial, googled.

**- How to verify it**
Follow the tutorial on a fresh VM.

**- Description for the changelog**

Added clearer instructions to the tutorial.

### CODE REVIEW

**Comment by rhatdan** [comment]
LGTM

**Comment by rhatdan** [comment]
No need for additional testing since this is only a documentation change.
@TomSweeneyRedHat @umohnani8 @mrunalp PTAL

File: tutorial.md [hunk 1]
```diff
@@ -192,8 +192,8 @@ RuntimeVersion:  1.10.0-dev
 RuntimeApiVersion:  v1alpha1
 ```
 
-> to avoid set --runtime-endpoint when call crictl,
-> you can export $CRI_RUNTIME_ENDPOINT=/var/run/crio/crio.sock
+> to avoid setting --runtime-endpoint when calling crictl,
+> you can export $CRI_RUNTIME_ENDPOINT=unix:///var/run/crio/crio.sock
```
Comment (umohnani8) [inline tutorial.md]: nit: "you can run `export...`"

File: tutorial.md [hunk 2]
```diff
@@ -183,7 +183,7 @@ sudo systemctl start crio
 #### Ensure the crio service is running
 
 ```
-sudo crictl --runtime-endpoint /var/run/crio/crio.sock info
+sudo crictl --runtime-endpoint unix:///var/run/crio/crio.sock
```
[the remark this answers is not in the document]
Comment (ktzee) [reply tutorial.md]: I replaced only the path. It worked for me while following the tutorial. I have no information about the "version" argument.

File: tutorial.md (hunk 1 above)
Comment (ktzee) [reply tutorial.md]: Fair enoug.

**Comment by ktzee** [comment]
I might have squashed a commit which is not mine into this PR...

**Comment by ktzee** [comment]
I think I made it... this was painful.

**Comment by rhatdan** [comment]
LGTM

**Comment by rhatdan** [comment]
@TomSweeneyRedHat @umohnani8 PTAL

**Comment by umohnani8** [comment]
LGTM

**Comment by rhatdan** [comment]
@ktzee Thanks
Merging since this is a docs only change.

**Pull request merged**